### PR TITLE
Add camel-github wrapper

### PR DESCRIPTION
**Pull request by stataru8**
camel-github wrapper

### CODE REVIEW

**Comment by essobedo** [comment]
There is a conflict to resolve

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -900,14 +900,12 @@
...
-<!--        &lt;!&ndash;the latest gson version of github core supports is 2.2.2 &ndash;&gt;-->
-<!--        <bundle dependency='true'>mvn:com.google.code.gson/gson/2.2.2</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.egit.github.core/${egit-github-core-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:commons-codec/commons-codec/${commons-codec-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-github/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-github' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <!-- the latest version of gson supported by github core is 2.2.2 -->
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.egit.github.core/2.1.5_1</bundle>
```
Comment (stataru8) [inline features/src/main/feature/camel-features.xml]: Wrap/Non-wrap for `org.eclipse.mylyn.github/org.eclipse.egit.github.core/2.1.5` bundle isn't working. Only the ServiceMix bundle is working.

TODO: change https://github.com/jbonofre/camel-karaf/blob/e5ec7577095193b9fbb65ef771017b45390b293a/pom.xml#L182

Comment (stataru8) [reply features/src/main/feature/camel-features.xml]: OK, I've seen that in this situation I should go with `${egit-github-core-version}_1`

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**